queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_part('year', date_of_birth) BETWEEN 2016 AND 2019;
+SELECT name FROM animals WHERE neutered IS TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Augmon', 'Pikachu');
```

Comment:
- [ ] This query result is not correct as you do not have ```Pikachu``` data

Reply:
Hello @Ghiftee 
I think I only forgot to include the SQL command of `Pikachu` in `data.sql` but it is there in the table, so the query works just fine, below is  a screenshot to show this. 
![Screenshot from 2023-01-16 14-29-00](https://user-images.githubusercontent.com/39535023/212668608-04cde4f3-8282-4b6c-91f2-6a79cf78a173.png)

#### Thank you